src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -47,9 +42,15 @@ final class CacheItem implements CacheItemInterface
     /**
      * @param mixed $value
      */
-    public function __construct(string $key, $value, bool $isHit, ?ClockInterface $clock = null)
-    {
-        $this->key   = $key;
+    public function __construct(
+        /**
+         * Cache key
+         */
```

Comment:
Can be dropped

Reply:
dropped at https://github.com/laminas/laminas-cache/pull/239/commits/cd455513e109d02a67c036a58791c657d7e6e702